### PR TITLE
dev-util/pycharm-professional Version 2017.3 bump bug 639758

### PR DESCRIPTION
**Pull request by StefanCristian**
Bug: https://bugs.gentoo.org/639758

Updates version to latest stable release

### CODE REVIEW

File: dev-util/pycharm-professional/pycharm-professional-2017.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils readme.gentoo-r1
```
Comment (jonasstein) [inline dev-util/pycharm-professional/pycharm-professional-2017.3.ebuild]: I think you need no eutils here https://blogs.gentoo.org/mgorny/2015/11/13/the-ultimate-guide-to-eapi-6/

File: dev-util/pycharm-professional/pycharm-professional-2017.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+}
+
+src_install() {
+	insinto /opt/${PN}
+	doins -r *
+
+	fperms a+x /opt/${PN}/bin/{pycharm.sh,fsnotifier{,64},inspect.sh}
+
+	dosym ../../opt/${PN}/bin/pycharm.sh /usr/bin/${PN}
+	newicon "bin/${MY_PN}.png" ${PN}.png
```
Comment (jonasstein) [inline dev-util/pycharm-professional/pycharm-professional-2017.3.ebuild]: if there are icons, and desktop files you likely need xdg functions
https://wiki.gentoo.org/wiki/Notes_on_ebuilds_with_GUI

**Comment by StefanCristian** [comment]
@jonasstein thanks a bunch for the observations, good thing we got this sorted out :)

**Comment by StefanCristian** [comment]
@jonasstein seems @xmw  hasn't approved anything since a very long time, is he still available?

**Comment by jonasstein** [comment]
In principle he is still alive, I saw him yesterday writing on bugzilla. He is also still merging to the tree https://dev.gentoo.org/~mgorny/dev-timeline.html 
How did you create the statistic/why do you think he  hasn't approved anything for a very long time?

**Comment by StefanCristian** [comment]
@jonasstein I meant particularly on github, but it's no problem. I was just wondering what might've gone wrong.

**Comment by StefanCristian** [comment]
@jonasstein my fixes were not ported to the new version yet. How should I proceed?
Open new bug for enhancement?

**Comment by xmw** [comment]
Hi, I've included the updates into both -community and -professional versions after reading
https://wiki.gentoo.org/wiki/Future_EAPI/Triggers#Current_trigger_candidates and realizing the need for it.
Personally, I've stopped using WMs supporting .desktop or icons nearly a century ago, so please apologize my ignorance.
FTR, I'm alive and well. And plan on responding to a lot of bugtraffic between the years ;-)

**Comment by StefanCristian** [comment]
@xmw ahahah. That's encouraging. Thanks a bunch, man. Have a nice one :)

**Pull request closed without merging**